lib/Sema/DerivedConformances.cpp

```diff
@@ -187,7 +191,10 @@ ValueDecl *DerivedConformance::getDerivableRequirement(TypeChecker &tc,
 
   // Local function that retrieves the requirement with the same name as
   // the provided requirement, but within the given known protocol.
-  auto getRequirement = [&](KnownProtocolKind kind) -> ValueDecl * {
+  // SWIFT_ENABLE_TENSORFLOW
+  auto getRequirement = [&](KnownProtocolKind kind,
+                            std::function<bool(ValueDecl *)> filter =
```

Comment:
llvm::function_ref

Reply:
Done in https://github.com/apple/swift/pull/25500/commits/fd11642607f33ea34851b27803686779f312940d.